smithy-smoke-test-traits/src/test/resources/software/amazon/smithy/smoketests/traits/errorfiles/vendor-params-validation.errors

```diff
@@ -0,0 +1,4 @@
+[WARNING] smithy.example#SayHello: Smoke test case defined a `vendorParamsShape` but no `vendorParams` | SmokeTestCase
```

Comment:
Would be great if this included the case ID

Reply:
Good call, done